about.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Where are the links to the css files. This page has no styling. You should at least link bootstrap css and a custom css file to customize the page